dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/trackedentity/DefaultTrackerOwnershipManager.java

```diff
@@ -349,14 +345,9 @@ private boolean hasTemporaryAccessWithUid(
 
     return tempOwnerCache.get(
         getTempOwnershipCacheKey(trackedEntityUid, program.getUid(), user.getUid()),
-        s -> {
-          TrackedEntity trackedEntity = trackedEntityService.getTrackedEntity(trackedEntityUid);
-          if (trackedEntity == null) {
```

Comment:
just to double-check if the te does not exist the count should be returned as 0, no NPEs right?

Reply:
Yes, this is just checking if it is present in the cache or not